styleguide/derek/examples/_data.json

```diff
@@ -15,8 +15,11 @@
   },
   "homepage": {
     "css": "css/derek.css"
-  }, 
+  },
   "forms": {
     "css": "css/derek.css"
+  },
+  "font-usage": {
```

Comment:
This is super helpful! One more ask: pretty please add a link to the font usage page to the examples page? 🍰